### PR TITLE
Loadouts - NATO Desert

### PR DESCRIPTION
**Pull request by SynixePac**

### CODE REVIEW

File: addons/loadouts/loadouts/NATO.hpp [hunk 1]
```diff
@@ -1,19 +1,9 @@
...
+        // Becomes: NATO Desert - AutoRifleman
+         AutoRifleman = QUOTE([["CUP_lmg_m249_pip1","muzzle_snds_m_snd_F","","CUP_optic_ElcanM145",["CUP_200Rnd_TE4_Green_Tracer_556x45_M249",200],[],""],[],["CUP_hgun_Glock17_tan","","acc_flashlight_pistol","",["CUP_17Rnd_9x19_glock17",17],[],""],["CFP_U_FieldUniform_marpat_d",[["ACRE_PRC343",1],["ACE_elasticBandage",3],["ACE_EarPlugs",1],["ACE_morphine",3],["ACE_packingBandage",3],["SmokeShell",1,1],["CUP_HandGrenade_M67",1,1]]],["SP_PlateCarrier1_Tan",[["CUP_200Rnd_TE4_Green_Tracer_556x45_M249",1,200],["MiniGrenade",1,1]]],["CFP_AssaultPack_MarpatDesert",[["CUP_200Rnd_TE4_Green_Tracer_556x45_M249",2,200]]],"CUP_H_USMC_MICH2000_DEF_ESS_DES","CUP_PMC_Facewrap_Tan",[],["ItemMap","","","ItemCompass","ItemWatch","CUP_NVG_PVS14"]])
```
Comment (BrettMayson) [inline addons/loadouts/loadouts/NATO.hpp]:
```suggestion
        AutoRifleman = QUOTE([["CUP_lmg_m249_pip1","muzzle_snds_m_snd_F","","CUP_optic_ElcanM145",["CUP_200Rnd_TE4_Green_Tracer_556x45_M249",200],[],""],[],["CUP_hgun_Glock17_tan","","acc_flashlight_pistol","",["CUP_17Rnd_9x19_glock17",17],[],""],["CFP_U_FieldUniform_marpat_d",[["ACRE_PRC343",1],["ACE_elasticBandage",3],["ACE_EarPlugs",1],["ACE_morphine",3],["ACE_packingBandage",3],["SmokeShell",1,1],["CUP_HandGrenade_M67",1,1]]],["SP_PlateCarrier1_Tan",[["CUP_200Rnd_TE4_Green_Tracer_556x45_M249",1,200],["MiniGrenade",1,1]]],["CFP_AssaultPack_MarpatDesert",[["CUP_200Rnd_TE4_Green_Tracer_556x45_M249",2,200]]],"CUP_H_USMC_MICH2000_DEF_ESS_DES","CUP_PMC_Facewrap_Tan",[],["ItemMap","","","ItemCompass","ItemWatch","CUP_NVG_PVS14"]])
```

**Review by BrettMayson — CHANGES_REQUESTED** [review]

**Comment by BrettMayson** [comment]
This should probably be named MARPAT Desert instead of just Desert

File: addons/loadouts/loadouts/NATO.hpp [hunk 2]
```diff
@@ -1,19 +1,9 @@
...
-        // Becomes: USMC Woodland - Squad Leader
-        Squad_Leader = QUOTE();
+class NATO {
+    class Desert {
+        // Becomes: NATO Desert - Squad Leader
+        Squad_Leader = QUOTE([["CUP_arifle_M4A1_BUIS_desert_GL","CUP_muzzle_snds_M16","","CUP_optic_ACOG",["CUP_30Rnd_556x45_PMAG_QP_Tan",30],[],""],[],["CUP_hgun_Glock17_tan","","acc_flashlight_pistol","",["CUP_17Rnd_9x19_glock17",17],[],""],["CFP_U_FieldUniform_marpat_d",[["ACRE_PRC343",1],["ACE_elasticBandage",3],["ACE_EarPlugs",1],["ACE_morphine",3],["ACE_packingBandage",3],["ACE_CableTie",5],["ACE_microDAGR",1],["MineDetector",1],["SmokeShell",1,1],["CUP_HandGrenade_M67",1,1]]],["SP_PlateCarrier1_Tan",[["ACRE_PRC148",1],["CUP_30Rnd_556x45_PMAG_QP_Tan",7,30],["CUP_30Rnd_556x45_PMAG_QP_Tracer_Green",1,30],["SmokeShellBlue",1,1],["SmokeShellRed",1,1],["1Rnd_HE_Grenade_shell",5,1]]],["CFP_AssaultPack_MarpatDesert",[["ACE_EntrenchingTool",1],["1Rnd_SmokeRed_Grenade_shell",1,1],["UGL_FlareGreen_F",1,1],["Laserbatteries",1,1]]],"CUP_H_USMC_MICH2000_DEF_ESS_DES","CUP_PMC_Facewrap_Tan",["Laserdesignator","","","",["Laserbatteries",1],[],""],["ItemMap","B_UavTerminal","","ItemCompass","ItemWatch","CUP_NVG_PVS14"]]);
```
Comment (BrettMayson) [inline addons/loadouts/loadouts/NATO.hpp]: - No vanilla mine detector
- No Packing Bandages
- No morphine
- Needs tourniquet

**Review by BrettMayson — CHANGES_REQUESTED** [review]

File: addons/loadouts/loadouts/NATO.hpp (hunk 2 above)
Comment (SynixePac) [reply addons/loadouts/loadouts/NATO.hpp]: -Removed Morphine from everyone Except Medic
-Gave out Tourniquets
-Replaced Vanilla Mine Detector with ACE
-Replaced Packing Bandages with Elastic

**Comment by BrettMayson** [comment]
Looks fine, I'll need to fix the ID'd radios before merge

**Review by BrettMayson — APPROVED** [review]

**Pull request merged**